packages/story-editor/src/components/footer/carousel/karma/carousel.karma.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma';
+
+describe('Carousel', () => {
```

Comment:
We also have a `carousel.karma.js` file in the canvas folder (search for `'Carousel integration'` to find it) -- should we perhaps use one file for carousel only?

Reply:
Rearranged in 0f873a462c. I was a bit surprised that we didn't have any tests actually...